sources/niLang/src/Expression.cpp

```diff
@@ -5807,8 +5807,8 @@ iExpressionVariable* Evaluator::CreateVariableFromVar(const achar* aaszName, con
         type = eExpressionVariableType_String;
       }
       else {
-        EXPRESSION_TRACE(niFmt("Invalid variant type: %s.", GetTypeString(varType)));
-        return NULL;
+        // Default to float
+        type = eExpressionVariableType_Float;
```

Comment:
This looks wrong to me. If an unsupported time is passed it it shouldn't silently go through.